styles.css

```diff
@@ -8,7 +8,12 @@
     position: relative;
     width: var(--menu-bar-width);
     height: var(--menu-bar-height);
-    margin: 100px;
+    transition: background-color 0.4s ease-in-out;
```

Comment:
Goed gedaan! Jouw versie werkt goed! Het viel me op dat de uitgewerkte versie van Nova die in de MS Teams link zat niet werkt (ik had die gedownload en in de browser geopend, er zitten bij haar uitwerking geen transitions in).